### PR TITLE
Add a second way of detecting whether DefTerm is available

### PR DESCRIPTION
**Pull request by DHowett**
;)

### CODE REVIEW

File: src/cascadia/TerminalSettingsModel/CascadiaSettings.cpp [hunk 1]
```diff
@@ -1087,7 +1087,21 @@ bool CascadiaSettings::IsDefaultTerminalAvailable() noexcept
...
 
-    return VerifyVersionInfoW(&osver, VER_BUILDNUMBER, dwlConditionMask) != FALSE;
+    if (VerifyVersionInfoW(&osver, VER_BUILDNUMBER, dwlConditionMask) != FALSE)
+    {
+        return true;
+    }
+
+    static bool isOtherwiseAvailable = [] {
+        wil::unique_hkey key;
+        const auto lResult = RegOpenKeyExW(HKEY_LOCAL_MACHINE,
```
Comment (lhecker) [inline src/cascadia/TerminalSettingsModel/CascadiaSettings.cpp:1097]: If you add a line break in front of `HKEY_LOCAL_MACHINE` the indentation will be a bit neater (or rather, more narrow).

BTW I just realized that we haven't updated clang-format in a while - it's still on v13, but VS already ships with v15! This means we can use `AlignAfterOpenBracket: BlockIndent` now!!

**Review by lhecker — APPROVED** [review]

**Review by zadjii-msft — APPROVED** [review]
(;

**Comment by DHowett** [comment]
Before I merge this, I'm going to get it tested.

**Comment by DHowett** [comment]
The detection works, so I'm merging this.

I have a followup PR that makes writing possibly work.

**Pull request merged**